app/views/pages/get-started/installing-nodejs-mac.liquid

```diff
@@ -0,0 +1,42 @@
+---
+converter: markdown
+metadata:
+  title: Installing Node.js on a Mac
+  description: This guide will help you install Node.js on a Mac.
+slug: get-started/installing-nodejs-mac
+searchable: true
+---
+
+This guide will help you install Node.js on a Mac.
+You will need NPM (Node.js Package Manager) to install the platformOS pos-cli, that allows you to easily deploy your configuration files and assets to our platform, and communicate with the platformOS API. NPM is distributed with Node.js: when you install Node.js, you automatically get NPM installed on your computer.
+
+## Steps
+
+Installing Node.js on a Mac is a two-step process:
+
+<div data-autosteps></div>
+
+### Step 1: Install Node.js
+
+Go to [nodejs.org](https://nodejs.org/en/). When visiting the page from a Mac, you will see download links for MacOS. Select the LTS (Long Term Support) version. The Current version has the latest features, but may be more prone to changes and bugs than the LTS version.
+
+When the download has finished, locate the doanloaded file in Finder and double-click on it. Go through the installation steps as instructed by the installer.
+
+When installation is complete, Node.js is ready to be used.
+
+### Step 2: Check if Node.js is installed
+
+To check if Node.js and NPM are installed, enter:
+
+* `node -v` in Terminal to see the version of Node.js installed
+* `npm -v` in Terminal to see the version of NPM installed
+
+The version number of Node.js/NPM will be displayed.
+
+To upgrade to the latest version of Node.js, enter `brew update` (to update the version of Node.js in Homebrew), and `brew upgrade node`(to upgrade Node.js).
```

Comment:
```suggestion
To upgrade to the latest version of Node.js, enter `brew update` (to update the version of Node.js in Homebrew), and `brew upgrade node`(to upgrade Node.js).
```

This is not applicable anymore. Upgrading is doing the same as installation, just with new installation file from nodejs.org

Reply:
Thank you! This is a leftover from the brew version, I will remove it.